src/pyggdrasil/tree_inference/_file_id.py

```diff
@@ -156,6 +172,52 @@ def _create_id(self) -> str:
     def __str__(self) -> str:
         return self.id
 
+    @classmethod
+    def from_str(cls, str_id: str):
+        """Creates a CellSimulation id from a string representation of the id.
+        Args:
+            str_id: str
+
+        Throws:
```

Comment:
I don't think a change needed here, but in the future I'd recommend `Raises` rather than `Throws` (so we can use automated documentation building utilities using Google style comments).